CleverPushLocation/Source/CleverPushLocation.m

```diff
@@ -7,6 +7,10 @@
 @implementation CleverPushLocation
 
 CLLocationManager* locationManager;
+NSTimer *geoFencetimer;
+double geoFencetimerValue;
+bool isCompletedGeoFence;
```

Comment:
What if we have multiple geoFences - is one global `bool` sufficient?

And please optimize naming.

Reply:
@gbhrdt Can we keep the name like 

geoFenceTimeoutCompleted || geoFenceTimeoutDone ?

Reply:
Yes sounds good

Do we need multiple bools or one is sufficient? I think we need multiple?

Reply:
I have tested with geofences array with multiple objects and it is working fine with multiple timers also.